resources/views/auth/registration_form.blade.php

```diff
@@ -0,0 +1,102 @@
+@extends('layout.app')
+@section('content')
+<header>
+    <div class="container flex_box">
+      <div class="header_title">
+        <h1>
+          <a href="../../index.html"><img src="{{ asset('/img/logo.png') }}" alt="ADSIST"></a>
+        </h1>
+      </div>
+      <div class="header_menu">
+        <ul class="flex_box">
+          <li class="hdmn_document"><a href="../../document/index.html">資料請求</a></li>
+          <li class="hdmn_login"><a href="{{ route('login') }}">ログイン</a></li>
+        </ul>
+      </div>
+    </div>
+  </header>
+  <main id="page_regist_form">
```

Comment:
we can set 
main tag with id from app.blade

app.blade
    <main id="@yield('page_id')">

other page usage:
@extends('layouts.app') 
@section('title', 'Registration') 
@section('page_id', 'page_regist_form')
@section('css', 'registration.css') 

@section('content') 
// your html here 
@endsection